src/style.css

```diff
@@ -1 +1,169 @@
-/* empty source */
+/* #region Normalization*/
+body {
+  margin: 0;
+  background: rgba(255, 255, 255, 1);
+}
+/*#endregion*/
+
+.mainCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+

```

Comment:
use this class for overall padding and also align items inside it to centre.
side wise it can be 16px
<img width="295" alt="image" src="https://user-images.githubusercontent.com/100297341/201100295-9888540a-dc47-44d0-a7b2-48f537ae95a9.png">

for the top it can be 32px
<img width="384" alt="image" src="https://user-images.githubusercontent.com/100297341/201100491-b0ab969a-4052-445b-9e51-18ca9c7a894c.png">

Reply:
plaese do these changes